proposals/function-pointers.md

```diff
@@ -11,122 +11,157 @@ potential implementation of the feature):
 
 https://github.com/dotnet/csharplang/issues/191
 
-This is an alternate design propsoal to [compiler intrinsics](https://github.com/dotnet/csharplang/blob/master/proposals/intrinsics.md)
+This is an alternate design propsoal to [compiler intrinsics]
+(https://github.com/dotnet/csharplang/blob/master/proposals/intrinsics.md)
 
 ## Detailed Design 
 
-### funcptr
-The language will allow for the declaration of function pointers using the `funcptr` contextual keyword. The 
-declaration and usage of function pointers closely resemble that of `delegate`:
+### Function pointers
+The language will allow for the declaration of function pointers using the `func*` syntax. The full syntax is described
```

Comment:
I very much preferred `funcptr`.

I would think you are also less likely to encounter something called `funcptr` than something called `func` and that `func*` could be ambiguous/confusing in some scenarios (is it a function pointer or a pointer to a struct called `func`... Just think of `func* func*() f`).

Reply:
I went for `func*` over `funcptr` because this is a pointer type fundamentally. Having a pointer type without the `*` token but also have it respect all pointer rules seems like an odd inconsistency in the language. 

There is no ambiguity in the syntax though. It may be confusing at a glance but the language can disambiguate.

Note: I fully expect LDM to spend at least an hour on syntax today though. Doubt the syntax will survive as is.